pkg/porter/lifecycle_test.go

```diff
@@ -87,7 +87,7 @@ func TestInstallFromTagIgnoresCurrentBundle(t *testing.T) {
 func TestBundleLifecycleOpts_ToActionArgs(t *testing.T) {
 	cxt := context.NewTestContext(t)
 
-	cxt.AddTestFile("testdata/install/base-params.txt", "base-params.txt")
+	cxt.AddTestFile("testdata/install/base-params.json", "base-params.json")
```

Comment:
This test file isn't used because reading the parameter set doesn't happen until the bundle is executed. Previously we would read the parameter file (with key value pairs) in this function, but now this isn't needed at all, we are just recording the name passed in.